attributes/default.rb

```diff
@@ -161,7 +165,7 @@
                                                 httpd boost-devel redhat-lsb mlocate mpich-devel openmpi-devel R atlas-devel
                                                 blas-devel fftw-devel libffi-devel openssl-devel dkms mysql-devel libedit-devel
                                                 libical-devel postgresql-devel postgresql-server sendmail mdadm python python-pip
-                                                libgcrypt-devel]
+                                                libgcrypt-devel libevent-devel]
```

Comment:
isn't this getting applied to centos 6 too?

Reply:
Yes. This is the package list for centos6 right?

Reply:
sorry just me misreading your update :D